docs/sphinx/users/comlinetools/formatlist.txt

```diff
@@ -2,34 +2,35 @@ List supported file formats
 ===========================
 
 A detailed list of supported formats can be displayed using the
-:command:`formatlist` command.
+:program:`formatlist` command.
 
-Current usage information can be shown by running:
+.. program:: formatlist
 
-::
+The default behavior is to print a plain-text list of formats::
 
-    formatlist -help
+    formatlist
 
-The default behavior is to print a plain-text list of formats:
+.. option:: -txt
 
-::
+    This can also be accomplished using by specifying the :option:`-txt`
+    argument::
```

Comment:
We're too distant from the previous sentence for the "This" to comfortably make immediate sense. Perhaps switch to follow the form of the other options, except with some additional brief note that it's the default? Not this PR's fault though.